src/praisonai/praisonai/cli/interactive/async_tui.py

```
@@ -197,12 +198,42 @@ def _get_agent(self):
+                max_chars = 32000  # Default cap
+                
+                # Check config for rules settings
+                try:
+                    from praisonai.cli.configuration.loader import load_config
+                    config = load_config()
+                    if config.rules.auto and not self.config.no_rules:
+                        should_load_rules = True
+                        max_chars = config.rules.max_chars
+                    elif self.config.no_rules:
+                        should_load_rules = False
+                except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)